src/pages/faq.tsx

```diff
@@ -0,0 +1,68 @@
+import { QuestionCard } from "@/components/cards/question-card";
+import { IconPencil } from "@tabler/icons-react";
+import Head from "next/head";
+
+const faq = [
+  {
+    question: "Can I edit my saves with stardew.app?",
+    answer:
+      "stardew.app is a perfection checklist, not a save file editor. While we allow you to upload your save files to the site, we do not store any more data than is needed to update you on your perfection status. We do not store your save file after upload and unfortunately cannot retrieve it for you.",
+    icon: IconPencil,
```

Comment:
You can probably just pull the icon out to the map. Seems unlikely there'd be a need for alternate icons.

Reply:
honestly thought it would be fun to have different questions distinguishing across the questions, but take a look at them and let me know